### ISSUE TITLE
Check for existence of data files before performing operations

### ISSUE THREAD

**Issue by comwrg** [plibither8/2048.cpp#12]
```
▶ ./build/2048.out

   /\\\\\\\\\          /\\\\\\\                /\\\         /\\\\\\\\\
  /\\\///////\\\      /\\\/////\\\            /\\\\\       /\\\///////\\\
  \///      \//\\\    /\\\    \//\\\         /\\\/\\\      \/\\\     \/\\\
             /\\\/    \/\\\     \/\\\       /\\\/\/\\\      \///\\\\\\\\\/
           /\\\//      \/\\\     \/\\\     /\\\/  \/\\\       /\\\///////\\\
         /\\\//         \/\\\     \/\\\   /\\\\\\\\\\\\\\\\   /\\\      \//\\\
        /\\\/            \//\\\    /\\\   \///////////\\\//   \//\\\      /\\\
        /\\\\\\\\\\\\\\\   \///\\\\\\\/              \/\\\      \///\\\\\\\\\/
        \///////////////      \///////                \///         \/////////



  Welcome to 2048!

          1. Play a New Game
          2. View Highscores and Statistics

  Enter Choice: 2

  SCOREBOARD
  ----------

  No saved scores.



  STATISTICS
  ----------

  +--------------------+------------+
  | Best Score         | 140732916466960 |
  | Game Count         |          1 |
  | Number of Wins     | -1837137648 |
  | Total Moves Played | 140734694615213 |
  | Total Duration (s) | 3.25939e-311 |
  +--------------------+------------+
```

**Comment by comwrg** [comment]
It also happens when you change the root directory of the run
```
~/Downloads/2048.cpp
▶ cd build

Downloads/2048.cpp/build
▶ ./2048.out

   /\\\\\\\\\          /\\\\\\\                /\\\         /\\\\\\\\\
  /\\\///////\\\      /\\\/////\\\            /\\\\\       /\\\///////\\\
  \///      \//\\\    /\\\    \//\\\         /\\\/\\\      \/\\\     \/\\\
             /\\\/    \/\\\     \/\\\       /\\\/\/\\\      \///\\\\\\\\\/
           /\\\//      \/\\\     \/\\\     /\\\/  \/\\\       /\\\///////\\\
         /\\\//         \/\\\     \/\\\   /\\\\\\\\\\\\\\\\   /\\\      \//\\\
        /\\\/            \//\\\    /\\\   \///////////\\\//   \//\\\      /\\\
        /\\\\\\\\\\\\\\\   \///\\\\\\\/              \/\\\      \///\\\\\\\\\/
        \///////////////      \///////                \///         \/////////



  Welcome to 2048!

          1. Play a New Game
          2. View Highscores and Statistics

  Enter Choice: 2

   /\\\\\\\\\          /\\\\\\\                /\\\         /\\\\\\\\\
  /\\\///////\\\      /\\\/////\\\            /\\\\\       /\\\///////\\\
  \///      \//\\\    /\\\    \//\\\         /\\\/\\\      \/\\\     \/\\\
             /\\\/    \/\\\     \/\\\       /\\\/\/\\\      \///\\\\\\\\\/
           /\\\//      \/\\\     \/\\\     /\\\/  \/\\\       /\\\///////\\\
         /\\\//         \/\\\     \/\\\   /\\\\\\\\\\\\\\\\   /\\\      \//\\\
        /\\\/            \//\\\    /\\\   \///////////\\\//   \//\\\      /\\\
        /\\\\\\\\\\\\\\\   \///\\\\\\\/              \/\\\      \///\\\\\\\\\/
        \///////////////      \///////                \///         \/////////



  SCOREBOARD
  ----------

  No saved scores.



  STATISTICS
  ----------

  +--------------------+------------+
  | Best Score         | 140732826948896 |
  | Game Count         |          1 |
  | Number of Wins     | -1837137648 |
  | Total Moves Played | 140734694615213 |
  | Total Duration (s) | 3.25939e-311 |
  +--------------------+------------+



  Press any key to exit:
```

**Comment by plibither8** [comment]
So there are two things for it to work properly (and show meaningful values):

1. The `data/` folder containing the `statistics.txt` and `scores.txt` files **needs** to be placed in the in the directory where the command is run from. The code is such that it looks for the `data` folder in the directory where the command was run from. This is a caveat and proper checks need to be implemented to ascertain first whether the files exist or not. I'll take this up.
2. Even if the files exist, if they are empty (in the first run, when you have not played any game) the program will not be able to assign values to existing variables, which ideally should have been assign to values in the file. Thus, garbage values are shown. 

Thanks for bringing this to notice, I'll work towards a solution.

**Comment by plibither8** [comment]
That's a great suggestion. I'll certainly implement it soon. Thanks a lot :smile:

**Comment by plibither8** [comment]
Fixed by #28
